.github/workflows/create-hotfix-branch.yml

```
@@ -16,7 +16,7 @@
     runs-on: ubuntu-latest
 
     # Only allow these users to create new hotfix branch from 'main'
-    if: github.ref == 'refs/heads/main' && (github.actor == 'vinayteki95' || github.actor == 'ItsSudip' || github.actor == 'krishna2020' || github.actor == 'koladilip' || github.actor == 'saikumarrs' || github.actor == 'sandeepdsvs' || github.actor == 'chandumlg' || github.actor == 'mihir-4116' || github.actor == 'utsabc' || github.actor == 'maheshkutty' || github.actor == 'manish339k') && (github.triggering_actor == 'ItsSudip' || github.triggering_actor == 'krishna2020' ||  github.triggering_actor == 'saikumarrs' || github.triggering_actor == 'sandeepdsvs' || github.triggering_actor == 'koladilip' || github.triggering_actor == 'chandumlg' || github.triggering_actor == 'mihir-4116' || github.triggering_actor == 'utsabc' || github.triggering_actor == 'vinayteki95' || github.triggering_actor == 'maheshkutty' || github.triggering_actor == 'manish339k')
+    if: github.ref == 'refs/heads/main' && (github.actor == 'vinayteki95' || github.actor == 'ItsSudip' || github.actor == 'krishna2020' || github.actor == 'koladilip' || github.actor == 'saikumarrs' || github.actor == 'sandeepdsvs' || github.actor == 'chandumlg' || github.actor == 'mihir-4116' || github.actor == 'utsabc' || github.actor == 'maheshkutty' || github.actor == 'manish339k' || || github.actor == 'abhimanyubabbar') && (github.triggering_actor == 'ItsSudip' || github.triggering_actor == 'krishna2020' ||  github.triggering_actor == 'saikumarrs' || github.triggering_actor == 'sandeepdsvs' || github.triggering_actor == 'koladilip' || github.triggering_actor == 'chandumlg' || github.triggering_actor == 'mihir-4116' || github.triggering_actor == 'utsabc' || github.triggering_actor == 'vinayteki95' || github.triggering_actor == 'maheshkutty' || github.triggering_actor == 'manish339k' || github.triggering_actor == 'abhimanyubabbar')
```

actionlint (1.7.7)

unexpected token "||" while parsing variable access, function call, null, bool, int, float or string. expecting "IDENT", "(", "INTEGER", "FLOAT", "STRING"

(expression)